completion/available/npm.completion.bash

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 # npm (Node Package Manager) completion
+# https://docs.npmjs.com/cli/completion
 
 if command -v npm &>/dev/null
 then
-  eval "$(npm completion)"
+  source <(npm completion)
```

Comment:
This does not work for me. When I use this line instead of the previous one that used `eval`, I don't have completion for `npm`.

Can you please check whether you got the syntax right for this one?